tools/ReleaseTool.Tests/ReleaseTool.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp2.1</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
@zeroZshadow do we want to pin the lang version in all of these too?

Reply:
Would be nice. I already saw that some were set to 7.3 and some 7.2

Reply:
+1 to pinning lang versions

Reply:
which version?

Reply:
Could stay on 7.3? If we need C#8 features we can update as/when needed?

Reply:
Equally 8 is fully backwards compatible with 7.3, so its not like choosing 8 has any downsides for us

Reply:
Yeah that's true